src/test/go_results.go

```diff
@@ -49,19 +49,41 @@ func parseGoTestResults(data []byte) (core.TestSuite, error) {
 					Stderr:   strings.Join(testOutput, ""),
 				})
 			} else if bytes.Equal(testResultMatches[1], []byte("SKIP")) {
-				i++ // Following line has the reason for being skipped
+				// The skip message is found at the bottom of the test output segment.
+				// Prior to Go 1.14 the test output segment follows the results line.
+				// In Go 1.14 the test output segment sits between the start line and the results line.
+
+				var outputLines []string
```

Comment:
Can you pull these five lines out to a function? It's repeated below (and in general I prefer a single-assignment rather than a declaration before an if this way, although that may be my old C++ days talking)